mlir/include/mlir/Dialect/LLVMIR/LLVMDialect.td

```diff
@@ -44,6 +44,7 @@ def LLVM_Dialect : Dialect {
     static StringRef getNoUndefAttrName() { return "llvm.noundef"; }
     static StringRef getDereferenceableAttrName() { return "llvm.dereferenceable"; }
     static StringRef getDereferenceableOrNullAttrName() { return "llvm.dereferenceable_or_null"; }
+    static StringRef getElementTypeAttrName() { return "llvm.element_type"; }
```

Comment:
```suggestion
    static StringRef getElementTypeAttrName() { return "llvm.elementtype"; }
```
nit: it seems like LLVM proper uses elementtype without underscore, so I would use the same convention here.